src/styles.ts

```diff
@@ -0,0 +1,56 @@
+import { H1 } from "@deskpro/deskpro-ui";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { Link } from "react-router-dom";
+import styled, { css } from "styled-components";
+
+export const StyledLink = styled(Link)`
```

Comment:
use `Link` from `app-sdk`

Reply:
I prefer the styling on this. Plus there's nothing wrong with using your component

Reply:
For what we created common ui components in the `app-sdk`?

Reply:
Just because we created doesnt mean we need to use it. Same thing with the Search component

Reply:
Refuses to redirect to the correct page.